README.md

```diff
@@ -75,3 +75,28 @@ curl http://localhost:8080/delete_order?id=2
 
 That's it. Feel free to fork this project and use it as a template for your own lightweight microservices!
 
+## Test with Docker
+
+Using a version of Docker with Wasm WASI support, start the example stack using `docker compose`:
```

Comment:
Can you provide a link on how to download and install the required Docker version? Thanks.

Reply:
I'm still getting that all setup but will do so when it's ready